src/core.ts

```diff
@@ -1,16 +1,34 @@
-import { IAddEventMessage, IBindingContainer, IClarityActivateErrorState, IClarityDuplicatedEventState, IClarityFields,
-  ICompressedBatchMessage, IEnvelope, IEvent, IEventArray, IEventBindingPair, IEventData,
-  IInstrumentationEventState, IMissingFeatureEventState, Instrumentation, IPayload, IPlugin, ITimestampedWorkerMessage,
-  ITriggerState, State, WorkerMessageType } from "../types/index";
-import compress from "./compress";
+import { IAddEventMessage, ICompressedBatchMessage, ITimestampedWorkerMessage, WorkerMessageType } from "../types/compressionworker";
+import {
+  IBindingContainer,
+  IClarityFields,
+  IEnvelope,
```

Comment:
Minor but why are some of these multiline and not others?

Reply:
Max length limitations. I turned the ones that didn't fit on a single line into multi-lines. Maybe it's not the best as far as consistency goes, i can change it to the original format like the "types/index" import that I removed above.

---
In reply to: [250243754](https://github.com/Microsoft/clarity-js/pull/123#discussion_r250243754) [](ancestors = 250243754)

Reply:
I personally prefer the multiple imported elements per line rather than 1 per line, otherwise we have an entire fold on my screen essentially of just imports

---
In reply to: [250326095](https://github.com/Microsoft/clarity-js/pull/123#discussion_r250326095) [](ancestors = 250326095,250243754)

Reply:
Yeah, ok, that makes sense

---
In reply to: [250243754](https://github.com/Microsoft/clarity-js/pull/123#discussion_r250243754) [](ancestors = 250243754)